### ISSUE TITLE
Do inline tables exist and do we need a $$TABLE tag for them?

### ISSUE THREAD

**Issue by m-k94** [DCMLab/ddd#39]
This is from Schröder, p. 28:

![InlineTableMultipleLines](https://user-images.githubusercontent.com/75686579/106364314-5f183700-632e-11eb-9059-56890387895d.png)

It's a rather strange thing when you try to segment it. I see it as two tables that are part of the same paragraph as what precedes them. They are introduced by "und zwar", followed by "unter a " and "unter b" respectively. So the tables are essentially elements of the same sentence and also not vertically separated from the text. 

Does this mean we have to also use a $$TABLE replacement-tag? It seems like something we'd probably like to avoid. However, in this specific case, I find it logically consistent, since it does appear to be in line to me.

(In the screenshot I still kept the contents of the tables in the transcription box because I was unsure.)

**Comment by fabianmoss** [comment]
You are right, these are technically, in-line tables. However, displaying them as such would probably distort the visual output a lot. 
I suggest in this and similar cases to treat them as Table elements AS IF they would break up the paragraph. This is not optimal but in this case I would rather opt for the pragmatic option. Can you please record this instance in the editorial decisions?
